csrc/multidevice/communicator.cpp

```diff
@@ -162,7 +162,7 @@ c10::intrusive_ptr<c10d::Backend> createBackend(
         store, static_cast<int>(rank), static_cast<int>(size), timeout);
   }
 #endif
-  NVF_ERROR(false, "no distributed backend available");
```

Comment:
I'm wondering if we would be better off keeping the error and reconfiguring the test to call createBackend after checking if the backend is available. Outside of the test suite it would be good to get an error message if we encounter this.

Reply:
Done.